mlflow/server/js/src/components/RunView.js

```diff
@@ -272,3 +313,15 @@ const shellEscape = (str) => {
   return str;
 };
 
+const styles = {
```

Comment:
Put the styles in the CSS if possible, not here, since it's confusing to split them across two types of files.